io.github.Youda008.DoomRunner.yaml

```diff
@@ -0,0 +1,96 @@
+app-id: io.github.Youda008.DoomRunner
+runtime: org.kde.Platform
+runtime-version: '5.15-22.08'
+sdk: org.kde.Sdk
+command: doomrunner.sh
+finish-args:
+  - --device=all
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --share=ipc
+  - --share=network
+  # Allow to execute host engines
+  - --talk-name=org.freedesktop.Flatpak
+  # Allow to access game files (eg .wad and .pk3)
+  - --filesystem=home:ro
```

Comment:
this remain a very broad filesystem permission

Reply:
Doom Runner it's not a game, it's a _launcher_. It combines an engine and game files to produce complex commands to run.
The author patched the original source ([see PR #109](https://github.com/Youda008/DoomRunner/pull/109)) in order to run engines by prefixing commands with `flatpak-spawn`.
The [README](https://github.com/mbugni/flathub/tree/io.github.Youda008.DoomRunner#readme) provides istructions to use both native and Flatpak engines.

#### Permissions explanation
`--device=all` is to use controllers.

`--talk-name=org.freedesktop.Flatpak` is to run engines.

`--filesystem=home:ro` is to choose game files.

#### Rationale
A tipical usage is to choose a combination of engines/files/options. Usually a gamer already has engines and files and wants a launcher to combine them to easily retrive configurations in subsequent gameplay.
If Doom Runner cannot launch engines, it's basically useless. A default engine has been provided to allow an "out of the box" experience, but players may want to use differente engines (like [io.github.fabiangreffrath.Doom](https://flathub.org/apps/io.github.fabiangreffrath.Doom), [com.zandronum.Zandronum](https://flathub.org/apps/com.zandronum.Zandronum) and so on...)